packages/Core/theme/tags.ts

```diff
@@ -116,12 +116,12 @@ export const getTags = (theme: WuiTheme): ThemeTags => {
       sm: {
         padding: `0 ${space.xxs}`,
         height: sizes.sm,
-        fontSize: fontSizes.meta2,
+        fontSize: fontSizes.sm,
```

Comment:
Isn't it the contrary ? meta2 => xs and meta1 => sm ?
(so you just have to switch line 119 & 124)

Reply:
Bien vu c'est corrigé :)